.changeset/strong-bobcats-notice.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/button': patch
+---
+
+- Buttons now correctly disable click events when disabled or isLoading
```

Comment:
Can we add a test for this please?

Reply:
There are already tests to make sure the `onClick` event isn't fired - the updates here are to make sure it doesn't LOOK like an event is fired. I'll update the changeset so it's accurate